### PR TITLE
pkg: fix compile errors in gecko_sdk and umorse

### PR DESCRIPTION
**Pull request by smlng**
### Contribution description

Currently all builds for boards with CPU emf32 fail with compiler error when using GCC 7.x.

This Pr disables the responsible Werror flag for the gecko sdk package.

### Issues/PRs references

see Murdock Log, e.g. [here](https://ci.riot-os.org/RIOT-OS/RIOT/8607/1fa97f0998b064e8ffcd11a7f442db67faecdbad/output/compile/tests/minimal/ikea-tradfri.txt)

### CODE REVIEW

**Comment by smlng** [comment]
🤞

**Comment by smlng** [comment]
added (intermediate) fix for umorse to fix missing (deprecated) definition of usleep function.

**Comment by kYc0o** [comment]
too fast!

**Comment by kYc0o** [comment]
is this last commit related?

**Comment by smlng** [comment]
> is this last commit related?

yes there was an error with GCC 7 and the umorse package, because the latter uses usleep. However, usleep is/was deprecated by POSIX a while ago and one should use nano sleep instead, additionally in the arm-none-eabi headers usleep is now guarded by macros and must be enable explicitly.

TL;DR: yes the last commit is related and needed.

**Comment by smlng** [comment]
I will fixed that upstream in umorse soon, but for now to keep murdock running, this fix is needed.

**Comment by kYc0o** [comment]
Yes, I agree with the fix, it's just that I was looking when it got merged and a new commit came "in the middle", thus I thought it was added after it was merged.

**Comment by kaspar030** [comment]
> I thought it was added after it was merged.

In this case, I saw it, ACKed it silently and then, when murdock agreed, merged. But thanks for watching!

**Comment by kYc0o** [comment]
Yeah I mean I was looking at it and suddenly a commit came and then merged (it took around 20min to get merged!), so that's why my surprise, but everything alright here :)
